pkg/kclient/generators.go

```diff
@@ -36,8 +36,7 @@ func GeneratePodTemplateSpec(objectMeta metav1.ObjectMeta, serviceAccountName st
 	podTemplateSpec := &corev1.PodTemplateSpec{
 		ObjectMeta: objectMeta,
 		Spec: corev1.PodSpec{
-			ServiceAccountName: serviceAccountName,
-			Containers:         containers,
+			Containers: containers,
```

Comment:
`GeneratePodTemplateSpec()` takes `serviceAccountName` but seems like we're not using it. Just curious, why was it removed? Do we not need it?

Reply:
I had removed it because we were previously hardcoding to `default` but that's the default in kube anyway and we don't have a way to specify a non-default service account at the moment so figured we can add it when we have thought about how users can customize it.